ale_linters/puppet/puppet.vim

```diff
@@ -1,5 +1,8 @@
 " Author: Alexander Olofsson <alexander.olofsson@liu.se>
 
+call ale#Set('puppet_puppet_executable', 'puppet')
+call ale#Set('puppet_puppet_options', '--color=false')
```

Comment:
We should probably take `--color=false` out of the configurable options here, and write that below. Otherwise people will be able to take the option away and make it so ALE can't parse the output.

Reply:
Yes. I initially coded it that way, but then somehow talked myself out of it. I’ll push another commit later tonight when I’m at a computer again.